sycl/doc/GetStartedGuide.md

```diff
@@ -493,8 +508,8 @@ int main() {
 
```

Comment:
Please update Table of Content with "Code the program for a specific GPU"

Reply:
Done in https://github.com/intel/llvm/pull/3177.